src/common/framework/edge.ml

```diff
@@ -21,9 +21,9 @@ type t =
     * transferred to the function node! *)
   | Test of CilType.Exp.t * bool
   (** The true-branch or false-branch of a conditional exp *)
-  | ASM of string list * asm_out * asm_in
+  | ASM of string list * asm_out * asm_in * bool
   (** Inline assembly statements, and the annotations for output and input
-    * variables. *)
+    * variables. The last field is a flag that indicates if this is a duplicated edge. *)
```

Comment:
You may add a few words to explain this in some more detail for future readers unaware of our discussions: Why are they duplicated and in what sense are they duplicated?